nikola/data/themes/bootstrap3/assets/css/theme.css

```diff
@@ -175,3 +175,24 @@ article.post-micro {
 .entry-summary {
   margin-top: 1em;
 }
+
+/* hat tip bootstrap/html5 boilerplate */
+@media print {
+    article .entry-title a[href]:after,
```

Comment:
Please send your complaints to [Bootstrap](https://github.com/twbs/bootstrap/blob/master/dist/css/bootstrap.css#L205) and [HTML 5 Boilerplate](https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css#L219) first.